examples/sampleview/core/sampledesigner/MultiLayerView.cpp

```diff
@@ -25,6 +25,11 @@
 #include <QStyleOptionGraphicsItem>
 
 namespace {
+    constexpr qreal width_multiplier = 1.15;
+    constexpr qreal multilayer_height = IView::basic_height;
+    constexpr qreal multilayer_width = IView::basic_width * width_multiplier;
+
+    constexpr QRectF defaultShape();
```

Comment:
At some point we have to extract this functionality to separate strategy class. Characteristic sizes should come from common (replaceable) source. This will help to handle HighDPI problem.